projects/helper/coreAssets.json

```diff
@@ -1259,7 +1259,8 @@
   },
   "starknet": {
     "STRK": "0x04718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d",
-    "DAI": "0x00da114221cb83fa859dbdb4c44beeaa0bb37c7537ad5ae66fe5e0efd20e6eb3",
+    "DAIv0": "0x00da114221cb83fa859dbdb4c44beeaa0bb37c7537ad5ae66fe5e0efd20e6eb3",
```

Comment:
please revert this change, it will break adapters relying on starknet.DAI pointing to old dai contract